src/include/parser/parsenodes.h

```diff
@@ -571,6 +571,12 @@ typedef struct DropStmt {
   bool concurrent;       /* drop index concurrently? */
 } DropStmt;
 
+typedef struct DropdbStmt {
```

Comment:
Maybe rename to `DropDatabaseStmt`?

Reply:
The reason I name this to DropdbStmt is because Create is named "CreatedbStmt". Do you want me to rename both?

Reply:
Yes, please rename both.